.changeset/gorgeous-rice-bow.md

```diff
@@ -0,0 +1,5 @@
+---
+"@kaizen/components": minor
```

Comment:
```suggestion
"@kaizen/components": patch
```
Nitpick: since the `Tag` is already published as a `minor`, this would technically be a `patch` that introduces no new functionality.